src/main/java/view/InputView.java

```diff
@@ -0,0 +1,10 @@
+package view;
+
+import java.util.List;
+
+public interface InputView {
+
+  List<String> inputCar();
```

Comment:
복수형, 입력받는 것에 대한 구체적인 이름 쓰기
```suggestion
  List<String> getCarNames();
```